jaeger-core/src/main/java/io/jaegertracing/JaegerSpan.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2016, Uber Technologies, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.jaegertracing;
+
+import io.opentracing.Span;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents a Span as seen from Jaeger's perspective. Builds on OpenTracing's {@link Span}, adding properties that
+ * are not part of the standard.
+ *
+ * Should be used wisely by instrumented applications: always prefer OpenTracing's approach when available.
+ */
+public interface JaegerSpan extends Span {
+
+  /**
+   * @return the underlying tracer
+   */
+  Tracer getTracer();
+
+  /**
+   * @return the timestamp in microseconds of when this span started
+   */
+  long getStart();
+
+  /**
+   * @return the Span's duration in microseconds
+   */
+  long getDuration();
+
+  /**
+   * @return a list of {@link Reference} for this span
+   */
+  List<Reference> getReferences();
+
+  /**
+   * @return an immutable map of tags that are part of this span
```

Comment:
It's not immutable at the moment - the content can change at any time.

Reply:
Perhaps `unmodifiable` would be a better term, as implementations should return a copy that cannot be modified by the caller (like, `return Collections.unmodifiableMap(tags);`)